megamek/build.gradle

```diff
@@ -243,21 +255,19 @@ distributions {
 createExe {
     description = 'Create Windows executable that wraps MM jar'
     mainClassName = application.mainClass
-    outfile = 'MegaMek.exe'
```

Comment:
This outfile assignment can be left in safely.   Was tested.

Reply:
Can be but not needed. It's default to project name plus exe.